src/components/AssetInfo/TokenSideBarContent.test.tsx

```diff
@@ -1,20 +1,25 @@
 import React from "react";
 import { mount } from "enzyme";
 import TokenSideBarContent from "./TokenSideBarContent";
+import TokenSideBarHolder from "./TokenSideBarHolder";
+import TokenSideBarBeneficiary from "./TokenSideBarBeneficiary";
 
 describe("tokenSideBarContent", () => {
-  it("should have 2 fields for Holder with no endorsement awaiting", () => {
-    const wrapper = mount(<TokenSideBarContent isHolder={true} />);
-    expect(wrapper.find("section")).toHaveLength(2);
+  it("should render <TokenSideBarHolder /> view", () => {
+    const wrapper = mount(
+      <TokenSideBarContent adminAddress="0xA" holderAddress="0xA" beneficiaryAddress="" approvedBeneficiaryAddress="" />
+    );
+    expect(
+      wrapper.containsMatchingElement(
+        <TokenSideBarHolder isEqualBeneficiaryAndHolder={false} approvedBeneficiaryAddress="" />
```

Comment:
Better to match the content rendered. if just want to test if component is there check for length. rest check what has been rendered from the component.